src/eko/evolution_operator/operator_matrix_element.py

```diff
@@ -241,7 +271,10 @@ def labels(self):
             logger.warning("%s: skipping non-singlet sector", self.log_label)
         else:
             labels.append((200, 200))
-            if self.is_intrinsic or self.backward_method is not None:
+            if (
+                self.is_intrinsic
```

Comment:
Do we already have a PR to drop this `intrinsic_range` once for all?

Reply:
#295